cupy/core/raw.pyx

```diff
@@ -218,7 +218,8 @@ cdef class RawModule:
     similarly.
 
     Args:
-        code_or_path (str): CUDA source code or path to cubin/ptx.
+        code (str): CUDA source code. Mutually exclusive with ``path``.
+        path (str): Path to cubin/ptx. Mutually exclusive with ``code``.
```

Comment:
Sorry @niboshi, one more question: are `code` and `path` really mutually exclusive here? Based on the changes I thought a user could specify both `code` and `path` without getting an error?

Reply:
Never mind, sorry for my stupidity 😔

Reply:
Sorry, I forgot to respond. Yes, I think they are mutually exclusive.